app/javascript/react/screens/App/common/ListViewToolbar/ListViewToolbar.js

```diff
@@ -9,12 +9,12 @@ import paginate from './paginate';
 class ListViewToolbar extends Component {
   state = {
     filterTypes: this.props.filterTypes,
-    currentFilterType: this.props.filterTypes[0],
+    currentFilterType: this.props.filterTypes[this.props.defaultFilterTypeIndex],
```

Comment:
Piggybacking on [comment](https://github.com/ManageIQ/manageiq-v2v/pull/729/files?utf8=%E2%9C%93&diff=split&w=1#r228152410) maybe we could do something like

```suggestion
    currentFilterType: this.props.defaultFilterTypeIndex ? this.props.filterTypes[this.props.defaultFilterTypeIndex] : this.props.filterTypes[0],
```

Reply:
Whoops, [defaultProps](https://github.com/ManageIQ/manageiq-v2v/pull/729/files#diff-3e23a8b28cd04fef009494fd9ca4aeddR233) already takes care of this.  Thanks for pointing that out @mturley!